src/lib/index.ts

```diff
@@ -199,18 +221,21 @@ class TPAStylePlugin {
           });
 
           const cssConfigFilename = generateStandaloneCssConfigFilename(file);
+          const defaults = this.getStyleParamsDefaultsContent(compilation.assets, file);
 
           const params = {
             cssVars,
             customSyntaxStrs,
             css,
             staticCss,
             compilationHash: this.compilationHash,
+            defaults: 'STYLE_PARAMS_DEFAULTS_PLACEHOLDER',
           };
 
           compilation.assets[cssConfigFilename] = this.generateStandaloneCssConfig({
             shouldEscapeContent,
             params,
+            defaults,
```

Comment:
i think it should be under `params`.

Reply:
For now I want to achieve that defaults would have raw function as a value, which will wrap the styleParams bundle.

If I will pass defaults value inside params here - the function will be a string (after JSON.stringify inside `generateStandaloneCssConfig`).

So I've added `defaults` to params with `STYLE_PARAMS_DEFAULTS_PLACEHOLDER` value. Lately inside `generateStandaloneCssConfig` I replacing `STYLE_PARAMS_DEFAULTS_PLACEHOLDER` with a string for defaults code, and in resulted bundle we will get raw function for `defaults` property value:
https://user-images.githubusercontent.com/439289/111795479-e8c89780-88cf-11eb-88ca-853cc37de9f4.png